ethcore/evm/src/interpreter/mod.rs

```diff
@@ -102,127 +103,234 @@ enum InstructionResult<Gas> {
 	StopExecution,
 }
 
+/// ActionParams without code, so that it can be feed into CodeReader.
+#[derive(Debug)]
+struct InterpreterParams {
```

Comment:
Why we need this? Couldn't we just use `ActionParams` and add some info that the code is always `None`?

Reply:
Actually on a second though, it's better to have it like that - prevents future errors of accessing the code.